pkg/devfile/adapters/common/command_simple.go

```diff
@@ -0,0 +1,99 @@
+package common
+
+import (
+	"fmt"
+	"github.com/openshift/odo/pkg/devfile/parser/data/common"
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/machineoutput"
+	"github.com/pkg/errors"
+)
+
+// simpleCommand is a command implementation for non-composite commands
+type simpleCommand struct {
+	info        ComponentInfo
+	adapter     commandExecutor
+	cmd         []string
+	id          string
+	component   string
+	originalCmd string
```

Comment:
Can you please add a comment to differentiate between originalCmd and cmd? Maybe rename cmd to something else.

Reply:
I've added some more details on the difference between both and what it means for `newSimpleCommand` vs `newOverriddenSimpleCommand`.